lib/src/commit_builder.rs

```diff
@@ -94,6 +94,13 @@ impl CommitBuilder<'_> {
         }
     }
 
+    pub fn reset_author(mut self, should_reset: bool) -> Self {
+        if should_reset {
+            self.commit.author = self.commit.committer.clone();
+        }
+        self
+    }
```

Comment:
nit: it's confusing that `reset_author(true)` is an immediate action, and `.reset_author(true).reset_author(false)` doesn't cancel the previous assignment.

I would add 0-ary `.reset_author()` (or just use `.set_author()`.)